nuget.config

```diff
@@ -3,5 +3,15 @@
   <packageSources>
     <clear />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+    <add key="azure-appservice" value="https://www.myget.org/F/azure-appservice/api/v3/index.json" />
```

Comment:
Is this temporary? There's a package on nuget but it isn't up to 36 yet...

Reply:
Yes, using the private nuget package until functions host is released with 36.

Reply:
No, the webjobs sdk with target based scaling support has not been released yet. Will update the package source as soon as it is released.

Reply:
Removed this